index.html

```diff
@@ -92,100 +171,184 @@ <h6>Skills</h6>
+                            </li>
+                            <li class="icons">
+                                <a href="#">
+                                    <img src="./assets/tw.png" alt="" />
+                                </a>
+                            </li>
+                            <li class="icons">
+                                <a href="">
+                                    <img src="./assets/m.png" alt="" />
+                                </a>
+                            </li>
+                        </ul>
+                        <p class="email">emailhere@email.com</p>
+                    </footer>
+                    
+                </div>
+            </section>
+
+            <section class="contact-mobile">
                 <h4>CONTACT US</h4>
                 <p class="contact-info">
                     If you have an application you are interested in developing, a
                     feature that you need built or a project that needs coding. I’d love
                     to help with it
                 </p>
-                <form action="https://formspree.io/f/mdoybpze" method="POST">
-                    <ul class="input-form">
-                        <li>
-                            <input name="name" required type="text" placeholder="Type your name" maxlength="30" />
-                        </li>
-                        <li>
-                            <input type="email" placeholder="Type your email" required name="email" />
-                        </li>
-                        <li>
-                            <textarea name="text" maxlength="500" id="input" cols="40" rows="10" required
-                                placeholder="Enter text here"></textarea>
-                        </li>
-                        <li>
-                            <button class="btn-btn" type="submit">Submit</button>
-                        </li>
-                    </ul>
-                </form>
+                <div class="form-info">
+
+                    <form action="https://formspree.io/f/mdoybpze" method="POST">
+                        <ul class="input-form">
+                            <li>
+                                <input name="name" required type="text" placeholder="Type your name" maxlength="30" />
+                            </li>
+                            <li>
+                                <input type="email" placeholder="Type your email" required name="email" />
+                            </li>
+                            <li>
+                                <textarea name="text" maxlength="500" id="input" cols="40" rows="10" required
+                                    placeholder="Enter text here"></textarea>
+                            </li>
+                            <li>
+                                <button class="btn-btn" type="submit">Submit</button>
+                            </li>
+                        </ul>
+                    </form>
```

Comment:
- [ ] Add all the necessary form fields for the desktop version of your app. Compare your page to the original.

<div class="flex">
<img src="https://user-images.githubusercontent.com/65501535/134750610-617b90c0-961d-4d39-9e30-b844805d377b.png"  style="width: 50%;"/>
<img src="https://user-images.githubusercontent.com/65501535/134750576-d03bf325-fab4-47e9-b89f-578a5363a379.png"  style="width: 50%;"/>
</div>
!

Reply:
Thank you. 
I am not sure if this was mention in the activity. IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.

Reply:
Okay, you may carry on with your design.

Reply:
Thank you.